test-adapter/src/main/scala/org/scalajs/testing/adapter/HTMLRunnerBuilder.scala

```diff
@@ -36,20 +36,24 @@ object HTMLRunnerBuilder {
       Files.createTempDirectory(outputPath.getParent(), ".html-artifacts")
 
     sys.addShutdownHook {
-      Files.walkFileTree(artifactsDir, new SimpleFileVisitor[Path] {
-        override def visitFile(file: Path, attrs: BasicFileAttributes): FileVisitResult = {
+      Files.walkFileTree(artifactsDir,
+          new SimpleFileVisitor[Path] {
+        override def visitFile(file: Path,
```

Comment:
Again: The nested scope is *less* indented than its parent. Is that really correct?